src/features/Send/common/SendContext.tsx

```diff
@@ -0,0 +1,222 @@
+import * as React from 'react'
+
+import {Address, Quantity, TokenId, YoroiTarget, YoroiUnsignedTx} from '../../../yoroi-wallets/types'
+
+export type SendState = {
+  selectedTargetIndex: number
+  selectedTokenId: TokenId
+  yoroiUnsignedTx: YoroiUnsignedTx | undefined
```

Comment:
i dont think `yoroiUnsignedTx` should be included in this hook
a sync hook to describe the transaction request
and an async hook to create the transaction